src/sonic-yang-models/setup.py

```diff
@@ -65,16 +61,26 @@ def run (self):
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
     ],
+    install_requires = [
+        'xmltodict==0.12.0',
+        'ijson==2.6.1'
+    ],
+    tests_require = [
+        'pytest>=3',
+        'ijson==2.6.1',
+        'pyang==2.1.1'
+    ],
+    setup_requires = [
+        'pytest-runner'
```

Comment:
Please also add 'wheel' to setup_requires for completeness.

Reply:
@jleveque : All changes done, Kindly review.